setup_tests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -o errexit
+
+if [[ -z "${MININORCAL_OSM_PATH}" || -z "${MININORCAL_GTFS_PATH}" ]]; then
+  echo "MININORCAL_OSM_PATH or MININORCAL_GTFS_PATH env vars are not set! Set these appropraitely and try again"
+  exit 1
+else
+  MININORCAL_OSM_PATH="${MININORCAL_OSM_PATH}"
+  MININORCAL_GTFS_PATH="${MININORCAL_GTFS_PATH}"
+fi
+
+echo "Checking if OSM + GTFS test files exist; downloading if needed"
+if [[ ! -f "./web/test-data/mini_nor_cal.osm.pbf" ]]; then
+  echo "Downloading OSM data for mini_nor_cal test region"
+  gsutil -m -o "GSUtil:parallel_process_count=1" cp $MININORCAL_OSM_PATH ./web/test-data/mini_nor_cal.osm.pbf
+  echo "Cutting down downloaded mini_nor_cal OSM to smaller cutout"
+  if ! command -v osmconvert; then
+    if [[ $OSTYPE == 'darwin'* ]]; then
+      brew install osmfilter
+    else
+      apt install -y osmctools
+    fi
+  fi
+  osmconvert ./web/test-data/mini_nor_cal.osm.pbf -b=-122.30986499194101,37.91724446910281,-120.68388843649487,39.42040570561121 --complete-ways --out-pbf > ./web/test-data/micro_nor_cal.osm.pbf
```

Comment:
micronit: maybe make the boundary points variables/constants at the top of the script for clarity?
I'd also add a comment about what this boundary is and why it was chosen.